src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -17,12 +18,19 @@ const GuideStore = Reflux.createStore({
       // The current step of the current guide (1-indexed). 0 if there's no guide
       // or the guide is just cued but not opened.
       currentStep: 0,
+
+      currentOrgSlug: null,
     };
     this.listenTo(GuideActions.fetchSucceeded, this.onFetchSucceeded);
     this.listenTo(GuideActions.closeGuide, this.onCloseGuide);
     this.listenTo(GuideActions.nextStep, this.onNextStep);
     this.listenTo(GuideActions.registerAnchor, this.onRegisterAnchor);
     this.listenTo(GuideActions.unregisterAnchor, this.onUnregisterAnchor);
+    this.listenTo(OrganizationsActions.setActive, this.onSetActiveOrganization);
```

Comment:
a) Does the active organization ever get unset? b) let's subscribe to `update` too (for org name changes) to be safe. should be super rare so not a big deal, but also might be super simple.

Reply:
It's not unset from OrganizationsActions—but that makes sense. In the event we're not sure what organization to redirect to, it's good to have the last active one to rely on.